src/jit/emitarm64.cpp

```diff
@@ -6220,6 +6335,102 @@ void emitter::emitIns_S_R(instruction ins, emitAttr attr, regNumber reg1, int va
 
 /*****************************************************************************
  *
+ *  Add an instruction referencing consecutive stack-based local variable slots and two registers
+ */
+void emitter::emitIns_S_S_R_R(
+    instruction ins, emitAttr attr1, emitAttr attr2, regNumber reg1, regNumber reg2, int varx, int offs)
+{
```

Comment:
Likewise:
Technically this method should be called emitIns_S_S_R_R_I because you require the immediate.